com.gluonhq.scenebuilder.yml

```diff
@@ -0,0 +1,48 @@
+app-id: com.gluonhq.scenebuilder
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: SceneBuilder
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+cleanup:
+  - /share/info/*
+  - /share/locale/*
+  - /share/man/*
+  - /share/runtime/locale/*
+  - 
+modules:
+  - name: SceneBuilder
+    buildsystem: simple
+    only-arches:
+    build-commands:
+    - mkdir -p deb-package
+    - ar p SceneBuilder-18.0.0.deb data.tar.xz | tar -xJf - -C deb-package
+    - mv deb-package/opt /app/lib 
+    - install -Dm644 com.gluonhq.scenebuilder.metainfo.xml -t /app/share/metainfo/
+    - install -Dm755 scenebuilder.sh /app/bin/SceneBuilder
+    - install -Dm644 com.gluonhq.scenebuilder.256.png /app/share/icons/hicolor/256x256/apps/com.gluonhq.scenebuilder.png
+    - install -Dm644 com.gluonhq.scenebuilder.128.png /app/share/icons/hicolor/128x128/apps/com.gluonhq.scenebuilder.png
+    - sed -i '/^Categories=/ d' /app/lib/scenebuilder/lib/scenebuilder-SceneBuilder.desktop 
+    - desktop-file-edit --set-icon=$FLATPAK_ID 
+      --add-category=Development
+      /app/lib/scenebuilder/lib/scenebuilder-SceneBuilder.desktop  
+    - desktop-file-install /app/lib/scenebuilder/lib/scenebuilder-SceneBuilder.desktop --dir=/app/share/applications
+
+    sources:
+    - type: file
+      filename: SceneBuilder.deb
+      url: https://github.com/gluonhq/scenebuilder/releases/download/18.0.0/SceneBuilder-18.0.0.deb
```

Comment:
Please add a stanza for https://github.com/flathub/flatpak-external-data-checker/ to automate updates.